scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
Again, why is it needed? This should also be covered by crosstool-ng already; it also creates a per-configuration buildtools dir in `.build/${CT_TARGET}/buildtools/bin` where it has shell wrappers like this:
````
avn@mistral:~/work/ctng/crosstool-ng$ cat .build/powerpc-405-linux-gnu/buildtools/bin/x86_64-build_pc-linux-gnu-gcc 
#!/bin/bash
exec '/usr/bin/gcc' "${@}"
````
From what I see in `scripts/crosstool-NG.sh.in`, it should be honoring the CT_BUILD_SUFFIX when it searches for the location of "gcc":
````
    # Now we have mangled our BUILD and HOST tuples, we must fake the new
    # cross-tools for those mangled tuples.
    CT_DoLog DEBUG "Making build system tools available"
    for m in ${install_build_tools_for}; do
        r="CT_REAL_${m}"
        v="CT_${m}"
        p="CT_${m}_PREFIX"
        s="CT_${m}_SUFFIX"
````
Why didn't this work for you?